features/step_definitions/menu_steps.rb

```diff
@@ -16,3 +16,22 @@
     FactoryGirl.create(:menu, title: menu[:title])
   end
 end
+
+Given(/^I am on the edit menu page for "([^"]*)"$/) do |menu_name|
+  @menu = Menu.find_by(title: menu_name)
+  visit edit_menu_path(@menu)
+end
+
+Given(/^I add a dish to the "([^"]*)" menu$/) do |menu|
+  menu_page = Menu.find_by(title: menu)
+  visit(edit_menu_path(menu_page))
+  steps %Q{
```

Comment:
Small q if you don't do any interpolation. On the other hand perhaps you should do just that. Hard coded values (like Pizza in this case) are usually not desirable.